### PR TITLE
BadRequest exception handling was added

### LINKED ISSUE #1890: BadRequestException: sample parameter size must fall in the range (1, 500)
webapi 2.9

### Expected behavior
The sample is successful generated or some human-readable error

### Actual behavior
image

`2021-06-02 07:45:23.438 ERROR 36 --- [nio-8080-exec-5] o.o.w.u.GenericExceptionMapper           : javax.ws.rs.BadRequestException: sample parameter size must fall in the range (1, 500)
	at org.ohdsi.webapi.cohortsample.dto.SampleParametersDTO.validate(SampleParametersDTO.java:39)
	at org.ohdsi.webapi.service.CohortSampleService.createCohortSample(CohortSampleService.java:131)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory$1.invoke(ResourceMethodInvocationHandlerFactory.java:81)`

### Steps to reproduce behavior
1)Generate any cohort
2)Samples
3)Enter the number of patients greater than 500

### PR DESCRIPTION
**Pull request by ssuvorov-fls**
Some translations were changed
fixes #1890

### CODE REVIEW

File: src/main/resources/i18n/messages_ko.json [hunk 1]
```diff
@@ -2621,7 +2621,7 @@
           "sampleNameEmpty": "샘플 이름은 비워 둘 수 없습니다",
           "numberOfPatients": "* 환자 수",
           "numberOfPatientsPlaceholder": "숫자 입력",
-          "numberOfPatientsEmpty": "환자 수는 양의 정수여야 합니다",
+          "numberOfPatientsEmpty": "Number of patients must be a positive integer between 1 and 500",
```
Comment (chrisknoll) [inline src/main/resources/i18n/messages_ko.json:2624]: Do we need a korean translation here?

File: src/main/resources/i18n/messages_zh.json [hunk 2]
```diff
@@ -2621,7 +2621,7 @@
           "sampleNameEmpty": "样本名称不能为空",
           "numberOfPatients": "*患者数量",
           "numberOfPatientsPlaceholder": "输入一个数字",
-          "numberOfPatientsEmpty": "患者数量必须正整数",
+          "numberOfPatientsEmpty": "Number of patients must be a positive integer between 1 and 500",
```
Comment (chrisknoll) [inline src/main/resources/i18n/messages_zh.json:2624]: Same here, should this be translated?

File: src/main/resources/i18n/messages_ko.json (hunk 1 above)
Comment (ssuvorov-fls) [reply src/main/resources/i18n/messages_ko.json:2624]: Yes, we need translations, but I can't get them now. Besides there're some other untranslated constants

File: src/main/resources/i18n/messages_zh.json (hunk 2 above)
Comment (ssuvorov-fls) [reply src/main/resources/i18n/messages_zh.json:2624]: Yes, we need translations, but I can't get them now. Besides there're some other untranslated constants

File: src/main/resources/i18n/messages_en.json [hunk 3]
```diff
@@ -2621,7 +2621,7 @@
           "sampleNameEmpty": "Sample name cannot be empty",
           "numberOfPatients": "* Number of patients",
```
Comment (anton-abushkevich) [inline src/main/resources/i18n/messages_en.json:2622]: Let's add text "(from 1 to 500)" to numberOfPatients. To all translations.

Comment (ssuvorov-fls) [reply src/main/resources/i18n/messages_en.json]: done

**Review by chrisknoll — APPROVED** [review]
Thanks for the information. This looks good to me, but defer to any other concerns that others raise.

**Pull request merged**